src/Neo/SmartContract/Manifest/ContractAbi.cs

```diff
@@ -62,10 +62,10 @@ public static ContractAbi FromJson(JObject json)
         {
             ContractAbi abi = new()
             {
-                Methods = ((JArray)json["methods"]).Select(u => ContractMethodDescriptor.FromJson((JObject)u)).ToArray(),
-                Events = ((JArray)json["events"]).Select(u => ContractEventDescriptor.FromJson((JObject)u)).ToArray()
+                Methods = ((JArray)json?["methods"])?.Select(u => ContractMethodDescriptor.FromJson((JObject)u)).ToArray(),
```

Comment:
Followed by a line of code 
`if (abi.Methods == null || abi.Methods.Length == 0) throw new FormatException();`
 I add `?` to avoid null reference exceptions. When adding unit tests, I found that this is prone to null reference exceptions